src/test/java/iog/psg/cardano/jpi/JpiResponseCheck.java

```diff
@@ -111,12 +114,24 @@ public static CardanoApi buildWithDummyApiExecutor() {
                             sync,
```

Comment:
I accidentally left 3 debug system.printlns in this class line 105. Could you remove please?

Reply:
sure np.